.coafile

```diff
@@ -0,0 +1,25 @@
+[default]
+bears = FilenameBear
+files = README.md, core/README.md, core/src/**.java, app/src/**.java,
+        java/src/**.java
+file_naming_convention = pascal
+
+[markdown]
+bears = MarkdownBear, LineLengthBear, SpaceConsistencyBear
+files = README.md, core/README.md
```

Comment:
wouldn't it be simpler to use `**.md` and define the `ignore` properly in the `default` section?